lib/model/QubitInformationObject.php

```diff
@@ -610,6 +611,24 @@ public function getDescendantsForExport($options = array())
     return $descendants;
   }
 
+  /* True if this information object or any descendant uses a LOD of $level. */
+  public function containsLevelOfDescription($level)
+  {
+    $sql = "
+      SELECT 1 FROM information_object
+      INNER JOIN term on term.id = information_object.level_of_description_id
```

Comment:
This join is unnecessary, can directly join on term_i18n:

INNER JOIN term_i18n on term_i18n.id = information_object.level_of_description_id